crates/witgen_macro_helper/src/lib.rs

```diff
@@ -14,10 +14,14 @@ pub fn parse_tokens(item: proc_macro2::TokenStream) -> Result<Wit> {
 }
 
 /// Read a crate starting from a single file then parse into Wit
-pub fn parse_crate_as_file(path: &Path) -> Result<Wit> {
+pub fn parse_crate_as_file(path: &Path) -> Result<File> {
     if let Ok(ast) = read_full_crate_source_code(path, |_| Ok(false)) {
-        Ok(ast.into())
+        Ok(ast)
```

Comment:
If it's a File then the name of the variable seems incorrect

Reply:
Touché